client/src/components/Checkout/Details.js

```diff
@@ -0,0 +1,54 @@
+import React from "react";
+import {
+  Grid,
+  Typography,
+  makeStyles,
+  Divider,
+  Button,
+  Box,
+} from "@material-ui/core";
+import { formatMoney } from "../../utils/utils";
+
+const useStyles = makeStyles((theme) => ({
+  description: {
+    marginTop: "1rem",
+  },
+  button: {
+    marginTop: "1rem",
+  },
+  buttonGroup: {
+    display: "flex",
+    flexDirection: "column",
+  },
+}));
+
+const Details = (props) => {
```

Comment:
Is this the same exact component from /item/Details? You can just import it from there